public/javascripts/jquery.tokeninput.js

```diff
@@ -214,24 +214,20 @@ $.TokenList = function (input, url_or_data, settings) {
     var hidden_input_describedby = $(input)
                                      .attr("aria-describedby");
 
-    // String used as the id attribute for the autocomplete dropdown
-    var autocomplete_dropdown_id = hidden_input_id + "_autocomplete_dropdown";
-
-    // Change the original label's for attribute to match the id attribute we give the new input box
+    // Change the original label's for attribute so it will match the id attribue we give the new input box
     hidden_input_label.attr({
-      'for': hidden_input_id + '_autocomplete',
-      'id': hidden_input_id + '_autocomplete_label'
+      'for': hidden_input_id + '_autocomplete'
     });
 
     // Give the new input box an id attribute based on the original input box's id
     // Originally included .css({outline: "none" }), but we actually want to see an outline for accessibility reasons
         var input_box = $("<input>")
         .attr({
           "aria-autocomplete": "list",
-          "aria-controls": autocomplete_dropdown_id,
+          "aria-expanded": "true",
           "autocomplete": "off",
           "class": "text",
-          "id": hidden_input_id + "_autocomplete",
```

Comment:
I think you need to keep this line, this might be why feature tests are failing like:

```
Unable to find field "Fandoms" that is not disabled (Capybara::ElementNotFound)
./features/step_definitions/work_steps.rb:369:in `/^I fill in basic work tags$/'
./features/step_definitions/work_steps.rb:13:in `/^I fill in the basic work information for "([^"]*)"$/'
./features/step_definitions/work_steps.rb:39:in `/^I set up (?:a|the) draft "([^"]*)"(?: with fandom "([^"]*)")?(?: with character "([^"]*)")?(?: with second character "([^"]*)")?(?: with freeform "([^"]*)")?(?: with second freeform "([^"]*)")?(?: with category "([^"]*)")?(?: with rating "([^\"]*)")?(?: (?:in|to) (?:the )?collection "([^"]*)")?(?: as a gift (?:for|to) "([^"]*)")?(?: as part of a series "([^"]*)")?(?: with relationship "([^"]*)")?(?: using the pseud "([^"]*)")?$/'
./features/step_definitions/work_steps.rb:74:in `/^I post (?:a|the) (?:(\d+) chapter )?work "([^"]*)"(?: with fandom "([^"]*)")?(?: with character "([^"]*)")?(?: with second character "([^"]*)")?(?: with freeform "([^"]*)")?(?: with second freeform "([^"]*)")?(?: with category "([^"]*)")?(?: with rating "([^\"]*)")?(?: (?:in|to) (?:the )?collection "([^"]*)")?(?: as a gift (?:for|to) "([^"]*)")?(?: as part of a series "([^"]*)")?(?: with relationship "([^"]*)")?(?: using the pseud "([^"]*)")?$/'
./features/step_definitions/work_steps.rb:151:in `/^I have a work "([^"]*)"$/'
./features/step_definitions/work_steps.rb:206:in `/^the work "([^"]*)"$/'
features/bookmarks/bookmark_create.feature:395:in `Given the work "Popular Work"'
```

I ran this command on your branch to view the diff for this file since the last release, and the ID removal stood out to me more:
```sh
git diff -w c8fe139... public/javascripts/jquery.tokeninput.js
```

Reply:
You're right -- thanks! I thought I was removing a different `id` attribute: https://github.com/otwcode/otwarchive/pull/3698/files#diff-1f19307fef8f8b79315ffd7b32658ed970ee5b7ac5ffb0b583a481296b9ed4daR223